assets/configuration/core_configuration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Do we want this to start matching the actual SDK version?

Reply:
Yes, I think so--but probably not today?